plugins/plugin-chart-echarts/src/BoxPlot/transformProps.ts

```diff
@@ -42,8 +48,8 @@ export default function transformProps(chartProps: ChartProps): EchartsProps {
   const numberFormatter = getNumberFormatter(numberFormat);
   const metricLabels = formdataMetrics.map(getMetricLabel);
 
-  const transformedData = data
-    .map(datum => {
+  const transformedData: (BoxplotDataItemOption & { name: string })[] = data
```

Comment:
I'm curious name isn't in the `BoxplotDataItemOption` interface/type. Can you check if it should be called something else? If not, let's keep the type unchanged and just `@ts-ignore` the line.